src/attacks/fgsm/fgsm.jl

```diff
@@ -7,11 +7,19 @@ using Flux: onehotbatch, onecold
 
 # White-box Fast Gradient Sign Method (FGSM) attack by Goodfellow et al. (arxiv.org/abs/1412.6572)
 # Code adapted from (github.com/jaypmorgan/Adversarial.jl)
-function FGSM(model, x, y; loss=cross_entropy_loss, ϵ = 0.3, min_label=0, max_label=9, clamp_range = (0, 1))
-    w, h, c = size(x)
```

Comment:
I have generalized this to ensure these attacks can also be used with non-image data (see test file)